app-install-banner/deferred-banner/readme.md

```diff
@@ -1,13 +1,13 @@
-App Install Banner User Response Sample
+Deferred App Install Banner Sample
 ===
 
-See https://googlechrome.github.io/samples/app-install-banner/useraction-banner/index.html for a live demo.
+See https://googlechrome.github.io/samples/app-install-banner/deferred-banner/index.html for a live demo.
 
 
 #### Directions:
 
-* Install Chrome, or another browser that supports Service Workers and manifest
-* Visit the site over two separate days.
-* Or, more usefully, enable chrome://flags/#bypass-app-banner-engagement-checks
+* Install Chrome, or another browser that supports Service Workers and manifest.
+* Wait a few minutes.
+* Or, enable chrome://flags/#bypass-app-banner-engagement-checks to enable the button right away.
 
-On the second vist, you will see a message on the page that says the user either installed the app or dismissed the banner.
\ No newline at end of file
+Once the button is enabled, Chrome will display the Add to Home Screen prompt when you click it.
```

Comment:
"Add to Home Screen" is for mobile. On Desktop, it's called "Install App". Maybe use "Chrome will trigger the install flow when you click it"?

Reply:
Updated these files to refer to "install flow"